src/atoms/weavemol.jl

```diff
@@ -1,13 +1,19 @@
 # some commentary
 
 # docstring...
+
+# represents a given pair of atoms by their respective indices in the adjacency matrix representation
+const PairIndexes = Tuple{Integer,Integer}
```

Comment:
I don't think we really need this. We can just directly store a graph in WeaveMol and then iterate over pairs that would need bond features using `edges(g)` and over pairs for pair features with something like `(i,j) if j<i for i,j in 1:nv(g)` (that syntax may not be quite right but you get the idea)

Reply:
Fixed in e4beb74.